stylesheet.css

```diff
@@ -0,0 +1,69 @@
+body {
+    background-color: #f1f1f1;
+    font-family: calibri;
+    color:#5f5f5f;
+}
+
+header {
+  background: #5f5f5f;
+    padding: 1em;
+}
+
+header a {
+    color: white;
+    margin: 2em;
+    text-decoration:none;
+    
+}
+h2 {
+    border-left: 22px;
+    border-left-color: aqua;
```

Comment:
Looks like you were on to something here, not sure exactly what. As you can see these styles don't do anything.

<img width="500" alt="Screenshot 2020-04-04 at 16 23 47" src="https://user-images.githubusercontent.com/13404719/78453273-df2f2b00-7690-11ea-8a44-249229f41a59.png">

If you want to add a border to the element's left side you would write it like

`border-left: 22px solid aqua;`

if you trying to use border to move the element around layout wise don't do that use margin and padding instead. 
https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/The_box_model